### PR TITLE
1.13 fixes aside from Aliases

### PR DESCRIPTION
**Pull request by TheLimeGlass**
Everything attempted to be complete with 1.9+

Description:
A fix for everything that is possible without touching aliases and anything related to items and their ID's

Also a small code clean and removal of un-needed code or un-used code. I can reply to anything if you have questions.

and idk how the travis failed, barely any changes were made.

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprBurnCookTime.java [hunk 1]
```diff
@@ -134,13 +135,18 @@ public void change(Event e, @Nullable Object[] delta, Changer.ChangeMode mode) {
 				break;
 			case SET:
 				value = (original) -> changed;
+			case DELETE:
+			case REMOVE_ALL:
+			case RESET:
+			default:
+				break;
```
Comment (Nicofisi) [inline src/main/java/ch/njol/skript/expressions/ExprBurnCookTime.java]: why?

**Comment by TheLimeGlass** [comment]
The rework was made almost the same time, do I just change it to that branch or will it delete this pull?

**Comment by TheBentoBox** [comment]
[Looks like you can change the target branch.](https://blog.github.com/2016-08-15-change-the-base-branch-of-a-pull-request/)

**Comment by bensku** [comment]
It seems that majority of changes, which change references to Material, would not work on 1.12 and below. Thus, I cannot merge this. Will probably manually pick the parts I can use at some point.

**Comment by Snow-Pyon** [comment]
@TheLimeGlass even if the name exist on 1.13 and lower, we decided that using aliases internally is the best choice to mantain compatibility between multiple versions. You can look at the EffEquip if you're interested on doing so.

**Comment by bensku** [comment]
Same material name in 1.12 and 1.13 does not guarantee actually same material. Everything follows Vanilla naming now, while it used not to do so; there may be some nasty surprises.

**Comment by TheLimeGlass** [comment]
Closing as Bensku has already done most of this stuff.

**Pull request closed without merging**